### PR TITLE
[WIP] Leverage the babel rest-object spread plugin

### PR DESCRIPTION
**Pull request by lavish205**
#304

### CODE REVIEW

**Comment by lavish205** [comment]
@n1k0 please review

**Comment by n1k0** [comment]
I don't see any use of object rest args spread, nor can see the Babel plugin installed.

File: src/collection.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ function createUUIDSchema() {
 }
 
 function markStatus(record, status) {
-  return Object.assign({}, record, {_status: status});
+  return Object.assign({}, record, _status: status);
```
Comment (leplatrem) [inline src/collection.js]: As described in the issue description, every `Object.assign()` should be removed

**Comment by leplatrem** [comment]
> please review

The tests should pass first :)

File: src/collection.js (hunk 1 above)
Comment (leplatrem) [inline src/collection.js]: See https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/

**Comment by leplatrem** [comment]
Closing, feel free to reopen if you come back to it :) Let us know if you need some help! Thanks Lavish!

**Pull request closed without merging**